src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +20,14 @@
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(s -> s.split(","))
+                .flatMap(Arrays::stream)
```

Comment:
```suggestion
		.flatMap(n -> Arrays.stream(n.split(",")))
```

we can combine map and flatMap into one flatMap